assets/js/atomic/blocks/product-elements/price/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface BlockAttributes {
+	productId?: number;
+	className?: string;
+	textAlign?: 'left' | 'center' | 'right';
+	isDescendentOfQueryLoop?: boolean;
```

Comment:
The attributes are optional to prevent a TS error in the following section:
- https://github.com/woocommerce/woocommerce-blocks/blob/04f3f4f2bdddae3babf0bab092f58488d109bedd/assets/js/blocks/cart/cart-cross-sells-product-list/cart-cross-sells-product.tsx#L52